Web/Edubase.Web.UI/Areas/Groups/Views/Group/CreateChildrensCentre.cshtml

```diff
@@ -185,14 +185,14 @@
                             </p>
                         </div>
                         <div class="govuk-grid-column-full">
-                            <h2 class="govuk-heading-m">New children's centre {{groupTypeName}} details</h2>
+                            <h2 id="new-childrens-centre-heading" class="govuk-heading-m">New children's centre {{groupTypeName}} details</h2>
                             <div class="warning-message" v-show="groupNameWarningMessage.length >0">
                                 <p v-html="groupNameWarningMessage"></p>
                             </div>
                             <dl class="govuk-summary-list">
                                 <div class="govuk-summary-list__row">
                                     <dt class="govuk-summary-list__key">Group name</dt>
-                                    <dd id="group-name" class="govuk-summary-list__value">{{groupName}}</dd>
+                                    <dd id="group-name-value" class="govuk-summary-list__value">{{groupName}}</dd>
```

Comment:
Observation: Convention on prior commits seem to be adding an `id` to both the name and value for `<dt>` and `<dd>` tags.

I defer to yourselves if it's needed here, but might suggest that other tests require it but for some reason the tests here do not (potential variations between what is being tested?)

Examples:

- https://github.com/DFE-Digital/get-information-about-schools/pull/261/files#diff-59135efa72ad3b96891acb2a2cc00b37c653f670089a4f46f300d9641793f928R26-R27

- https://github.com/DFE-Digital/get-information-about-schools/pull/255/files#diff-2d20019a404cda40a221780418790385660f45e1fb62716ad6324d3978cf66e4R60-R61

Reply:
Change pushed @RogerHowellDfE if you could resolve and approve if happy.

Reply:
Sorry, thought I'd done it earlier since this was more a "do whatever you need to do, just an observation about an inconsistency"! 😁 

Done now 👍